### PR TITLE
Cell and row styling including class rules

### PR DESCRIPTION
**Pull request by BSd3v**
- added support for `cellClassRules`, `rowClassRules` and `getRowClass`
- added tests to check for each class rule under `test_class_rules.py`
- adjusted test for conditional formatting to include row conditional formatting

### CODE REVIEW

File: tests/test_class_rules.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            dag.AgGrid(
+                id="grid",
+                columnDefs=columnDefs,
+                rowData=rowData,
+                defaultColDef={
+                    "editable": True,
+                },
+                rowClassRules={"orange": "params.data.make == 'Ford'",
+                               "red": "params.data.make == 'Porsche'",
+                               "blue": "params.data.make == 'Toyota'"},
```
Comment (alexcjohnson) [inline tests/test_class_rules.py:44]: For clarity, can we give the classes in `rowClassRules` different names & functions from the ones in `columnDefs`?

Comment (BSd3v) [reply tests/test_class_rules.py:44]: Sure, I'm adding `row-` in front of them.

Comment (BSd3v) [reply tests/test_class_rules.py]: Also made the functions query the model instead of the make

Comment (BSd3v) [reply tests/test_class_rules.py]: @alexcjohnson :)

**Review by alexcjohnson — APPROVED** [review]
💃 Great!

**Pull request merged**